ws-nextjs-app/cypress/support/helpers/runTestsForPage.ts

```diff
@@ -30,11 +31,13 @@ export default ({
             beforeAll.forEach(runBeforeAll => runBeforeAll());
 
             // Ensure that the page is returning a 200 response code
-            cy.testResponseCodeAndRetry({
-              url: path,
-            });
+            if (failOnStatusCode) {
+              cy.testResponseCodeAndRetry({
+                url: path,
+              });
+            }
```

Comment:
I think there's a better way of fixing this - 
```suggestion
              cy.testResponseCodeAndRetry({
                url: path,
                responseCode: failOnStatusCode ? 200 : 404,
              });
```

Reply:
tried this way but all tests failed.

<img width="933" height="834" alt="image" src="https://github.com/user-attachments/assets/4a2f1b92-06cd-49ce-98f8-73027c14157b" />

Reply:
OK let's put it back the way it was - my apologies!

Reply:
No worries - I only pushed the other changes and kept this one as is.